qcengine/programs/tests/test_molpro.py

```diff
@@ -27,20 +28,36 @@ def test_molpro_output_parser(test_case):
     output_ref = qcel.models.Result.parse_raw(data["output.json"]).dict()
     output_ref.pop("provenance", None)
 
-    # TODO add `skip` to compare_recusive
+    # TODO add `skip` to compare_recursive
```

Comment:
`compare_recursive` takes a `forgive` list where it won't fail even if missing or mismatch.

Reply:
I don't think I actually put the `skip` comment there and I'm unsure of what we are worried about not matching.